lib/fizzy/execute.cpp

```diff
@@ -691,7 +693,7 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, span<const Value>
             const auto br_table_size = read<uint32_t>(immediates);
             const auto arity = read<uint32_t>(immediates);
 
-            const auto br_table_idx = stack.pop();
+            const auto br_table_idx = stack.pop().as<uint32_t>();
```

Comment:
Make sure these functional changes are in a separate commit.

Reply:
Oh, the _idx stuff is a separate commit, also available as #461.